langs/id.json

```diff
@@ -0,0 +1,189 @@
+{
+  "locale": "id",
+  "languages": {
+    "ab": "Abkhaz",
```

Comment:
for consistency with the existing language files, please have them order by language code